src/restorer/mnemonic-restorer.ts

```diff
@@ -22,36 +22,42 @@ function restorerFromEsplora<R extends MasterPublicKey>(
     gapLimit = 20,
   }) => {
     const restoreFunc = async function(
```

Comment:
could be cool to write a test for restorerFromState with an empty seed

Reply:
@Janaka-Steph